microsoft_atp/komand_microsoft_atp/actions/get_missing_software_updates/action.py

```diff
@@ -0,0 +1,22 @@
+import insightconnect_plugin_runtime
+from .schema import GetMissingSoftwareUpdatesInput, GetMissingSoftwareUpdatesOutput, Input, Output, Component
+# Custom imports below
+
+
+class GetMissingSoftwareUpdates(insightconnect_plugin_runtime.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='get_missing_software_updates',
+                description=Component.DESCRIPTION,
+                input=GetMissingSoftwareUpdatesInput(),
+                output=GetMissingSoftwareUpdatesOutput())
+
+    def run(self, params={}):
+        machine_id = self.connection.client.find_first_machine(params.get(Input.MACHINE)).get("id")
+        self.logger.info("Attempting to isolate machine id: " + machine_id)
```

Comment:
This log line doesn't make sense in this action. It sounds like we're trying to Isolate/Quarantine instead of getting software updates

```suggestion
        self.logger.info("Attempting to isolate machine id: " + machine_id)
```